cdap-docs/examples-manual/source/examples/sport-results.rst

```diff
@@ -0,0 +1,212 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform WordCount Application
+    :copyright: Copyright © 2014-2015 Cask Data, Inc.
+
+.. _examples-fileset:
+
+=============
+Sport Results
+=============
+
+A Cask Data Application Platform (CDAP) Example demonstrating partitioned file sets through sport results analytics.
+
+Overview
+========
+
+This Application demonstrates the use of the PartitionedFileSet dataset:
+
+   - Game results are stored in the PartitionedFileSet ``results``. It is partitioned by league and season,
+     and each partition is a CSV (comma-separated values) file containing the results in one league for a season;
+     for example, the 2014 season of the NFL (National Football League).
+   - Results are uploaded into the file set using the ``UploadService``.
+   - The ``ScoreCounter`` MapReduce program reads game results for a given league and
+     aggregates total counts such as games won and lost, or points scored and conceded, across all
+     seasons, and writes these totals to the partitioned file set ``totals`` that is partitioned by league.
+   - Both the original game results and the aggregated totals can be explored using ad-hoc SQL queries.
+
+Let's look at some of these components, and then run the Application and see the results.
+
+The SportResults Application
+----------------------------
+
+As in the other :ref:`examples,<examples-index>` the components
```

Comment:
This was copied from an existing example... better check everywhere else, too. Found 4 more and fixed.